determined-ai/determined
# build: INFENG-743: remove Helm bumpversion

## Caveat: this depends on #9661. That pull request should be reviewed and merged first.
## Ticket
INFENG-743

## Description
This changeset removes the bumpversion dependency from the Determined Helm chart. It is straightforward: we replace the `Chart.yaml` version strings with dummy values, and then pass version strings dynamically at build time via `--version` and `--app-version` flags to `helm package`.

## Test Plan
1. `cd` into `helm`.
1. Run `make build`.
1. Inside the `build` subdirectory, you should see an artifact like `determined-0.32.1+2dab06d0e.tgz`.
1. Run `tar -xzvf determined-0.32.1+2dab06d0e.tgz`, then `cd` into `determined`. Inside, you should find a `Chart.yaml` file with `0.32.1+2dab06d0e` in the `version` and `appVersion` keys.
1. Run `make clean` under `helm/` when you're finished.

## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
